requirements/base.txt

```diff
@@ -12,4 +12,3 @@ celery==3.1.23
 tqdm==4.8.3                     # progress bars
 requests==2.10.0
 https://github.com/cherrypy/cherrypy/archive/v6.2.0.zip
```

Comment:
I don't think `mock` is part of Python 2's standard library (it is for Python 3.3). Should we add it here?